tdrs-backend/plg/grafana_views/user_views/tribal_tanf_t3.sql

```diff
@@ -0,0 +1,125 @@
+                    EXTRACT(YEAR FROM CURRENT_DATE) OR
+                    -- Validate month (01-12)
+                    CAST(SUBSTRING("DATE_OF_BIRTH" FROM 5 FOR 2) AS INTEGER) NOT BETWEEN 1 AND 12 OR
+                    -- Validate day (01-31)
+                    CAST(SUBSTRING("DATE_OF_BIRTH" FROM 7 FOR 2) AS INTEGER) NOT BETWEEN 1 AND 31
+            THEN 0
+            ELSE 1
+        END AS "AGE_VALID",md5("SSN"::text) as "SSN_HASH",CASE
+            WHEN "SSN" !~ '^(1{9}|2{9}|3{9}|4{9}|5{9}|6{9}|7{9}|8{9}|9{9})$' THEN 1
+            ELSE 0
+        END AS "SSN_VALID","RACE_HISPANIC","RACE_AMER_INDIAN","RACE_ASIAN","RACE_BLACK","RACE_HAWAIIAN","RACE_WHITE","SEX","RECEIVE_NONSSA_BENEFITS","RECEIVE_SSI","RELATIONSHIP_HOH","PARENT_MINOR_CHILD","EDUCATION_LEVEL","CITIZENSHIP_STATUS","UNEARNED_SSI","OTHER_UNEARNED_INCOME", 
+    data_files.section,
+    data_files.version,
+    data_files.year,
+    data_files.quarter,
+    stt.name AS "STT",                                                         -- Select stt_name from the stts table
+    stt.stt_code AS "STT_CODE",                                                -- Select stt_code from the stts table
+    stt.region_id AS "REGION"                                                  -- Select region from the stts table
+FROM search_indexes_TRIBAL_TANF_T3 T3
+INNER JOIN
+        data_files_datafile data_files                                             -- Join with data_files_datafile
+        ON T3.datafile_id = data_files.id                              -- Join condition
+    INNER JOIN (
+        SELECT
+            stt_id,                                                                -- Select stt_id
+            section,                                                               -- Select section
+            year,                                                                  -- Select fiscal_year
+            quarter,                                                               -- Select fiscal_quarter
+            MAX(version) AS version                                                -- Get the maximum version for each group
+        FROM
+            data_files_datafile                                                    -- Subquery table
+        GROUP BY
+            stt_id, section, year, quarter                                         -- Group by columns
+    ) most_recent
+        ON data_files.stt_id = most_recent.stt_id
+        AND data_files.section = most_recent.section
+        AND data_files.version = most_recent.version
+        AND data_files.year = most_recent.year
+        AND data_files.quarter = most_recent.quarter
+    INNER JOIN
+        stts_stt stt                                                               -- Join with the stts table (aliased as stt)
+        ON data_files.stt_id = stt.id                                              -- Join condition to match stt_id
+    WHERE
```

Comment:
noting here as a reminder to self for another ticket: tdp is currently storing 2nd child records that are incomplete. this is a result of stts submitting malformed tanf/tribal t3 and ssp m3  records. we'll want to filter these out of the views. this is feasible to handle as part of dashboard creation for now :)